programming-core/src/main/java/org/objectweb/proactive/core/httpserver/ClassServerServlet.java

```diff
@@ -132,7 +132,10 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
         if (logger.isDebugEnabled()) {
             final String from = req.getLocalAddr() + ":" + req.getLocalPort();
             final String status = ok ? "OK" : "Class Not Found";
-            logger.debug("Served request from " + from + " for " + className + " status: " + status);
+
+            logger.debug("Served request from " + from + " for " + className + " status: " + status +
+                         (bytes != null ? " bytes: " + bytes.length : ""));
```

Comment:
You could replace `bytes != null ? " bytes: " + bytes.length : "")` with something like `bytes != null ? " bytes: " + bytes.length : "bytes array null")`

That could prevent having empty lines in the logs.

Reply:
Don't get your point